packages/test/package.json

```diff
@@ -25,18 +25,36 @@
   "author": "Temporal Technologies Inc. <sdk@temporal.io>",
   "license": "MIT",
   "dependencies": {
+    "@grpc/grpc-js": "^1.6.8",
+    "@grpc/proto-loader": "^0.7.0",
+    "@opentelemetry/api": "^1.1.0",
+    "@opentelemetry/core": "^1.4.0",
     "@opentelemetry/exporter-trace-otlp-grpc": "^0.29.2",
+    "@opentelemetry/sdk-node": "^0.30.0",
+    "@opentelemetry/semantic-conventions": "^1.4.0",
     "@temporalio/activity": "file:../activity",
     "@temporalio/client": "file:../client",
     "@temporalio/common": "file:../common",
+    "@temporalio/core-bridge": "^1.0.0-rc.1",
     "@temporalio/interceptors-opentelemetry": "file:../interceptors-opentelemetry",
     "@temporalio/proto": "file:../proto",
     "@temporalio/testing": "file:../testing",
     "@temporalio/worker": "file:../worker",
     "@temporalio/workflow": "file:../workflow",
     "@types/async-retry": "^1.4.4",
+    "@types/dedent": "^0.7.0",
```

Comment:
`@types/*` should be `devDependencies`. Note that `ava` has to be kept in `dependencies` for this project.